lib/SubscriptionsManager.js

```diff
@@ -54,9 +55,39 @@ SubscriptionsManager.prototype.set = function (entryId, email) {
     return this._get(entryId).then(list => {
       if (!list) {
         queue.push(new Promise((resolve, reject) => {
-          this.mailAgent.lists().create({
+          let payload = {
```

Comment:
```suggestion
          let newMailingListConfig = {
```
This name would be more descriptive